src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2338,8 +2371,12 @@ private void flushReplyWindow(
                 {
                     if (isReplyOpen())
                     {
+                        final int slotCapacity = bufferPool.slotCapacity();
+                        final int maxNumberOfFrames = slotCapacity / initialMaxFrameSize +
+                            ((slotCapacity % initialMaxFrameSize == 0) ? 0 : 1);
```

Comment:
Try to avoid `%` operator as it is quite heavy.
Does `final int maxNumberOfFrames = (slotCapacity + initialMaxFrameSize - 1) / initialMaxFrameSize;` work?